cg/meta/archive/ddn/utils.py

```diff
@@ -0,0 +1,30 @@
+from cg.meta.archive.ddn.constants import (
+    METADATA_FIELD_NAME,
+    METADATA_FIELD_VALUE,
+    MetadataFields,
+)
+from cg.store.models import Sample
+
+
+def get_metadata(sample: Sample) -> list[dict]:
+    """Returns metadata generated from a sample."""
+    metadata: list[dict] = [
+        {
+            METADATA_FIELD_NAME: MetadataFields.CUSTOMER_NAME.value,
```

Comment:
```suggestion
            METADATA_FIELD_NAME: MetadataFields.CUSTOMER_NAME,
```
Should work with StrEnum

Reply:
Already tried it with that and the request did not go through 😞